### DISCUSSION TITLE
Feature Request: Option to keep Panel Query and Alert Query in Sync

### DISCUSSION THREAD

**Discussion by stefankaufmann** [grafana/grafana#55146] [Alerting]
Hi, 

After the upgrade to Grafana 9.1, the alert query seems to be decoupled from the query of the panel. We understand, that this can be quite useful. However, when we want to keep the panel query and the alert query in sync, we manually need to adjust both queries. 

The usability for keeping panel and alert query in sync was much better (and also more intuitive) in grafana 8.5

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
